### PR TITLE
chore(doc): suspend exercises

### PR DESCRIPTION
**Pull request by ggutoski**
remove exercises and change doc heading to "Exercises (coming soon)".

### CODE REVIEW

**Comment by Windforcer** [comment]
Are we removing ex 4 and ex5 as well?

File: documentation/docs/Exercises/_category_.json
Comment (Windforcer) [inline documentation/docs/Exercises/_category_.json:4]: Missing newline

**Comment by Windforcer** [comment]
Failed deployment, likely due to broken links now that the exercises are removed. Other Vercel pages probably linked to some of these exercises. You can control F and search for the slugs removed (eg, /exercises/e1)

**Review by Windforcer — APPROVED** [review]

**Comment by Windforcer** [comment]
tentatively approved for now :) once links and newline is fixed, we can merge

File: documentation/docs/Exercises/_category_.json
Comment (ggutoski) [reply documentation/docs/Exercises/_category_.json:4]: My IDE keeps stripping away the newline.  I'll add it via this github comment 😛 
```suggestion
}

```

**Comment by Windforcer** [comment]
After discussion, we want the team's opinion on whether or not to remove ex 1-3 or mark their status as inactive. The goal is to not confuse testnet users. Is it more confusing visit exercises page and see it starts on ex 4? or is it more confusing to click into ex 1 then see it is inactive, or perhaps even complete the exercise having missed that it was inactive?

we should keep ex 4 and ex 5 for at least another testnet cycle since we did not tell users they would be phased out.

**Pull request closed without merging**
